test/velociprobe.py

```diff
@@ -0,0 +1,18 @@
+"""Test conversion from velociprobe data to nexus.
```

Comment:
Where is the _testing_ done?  This generates a NeXus file with specific imports.

Reply:
There's no format validator yet, so this "test" just checks that the converter executes without error.

Reply:
Could you refactor to use pytest and assert that the code does not raise an exception?

Reply:
This could be the beginning of implementing deeper tests as desired.

Reply:
Converted examples to testing framework. Internet says that the default assumption is "no exception", so I don't need to "assert no exception".